proto/elys/clob/query.proto

```diff
@@ -27,16 +28,66 @@ service Query {
   }
 
   rpc SubAccounts(SubAccountsRequest) returns (SubAccountsResponse) {
-    option (google.api.http).get = "/elys-network/elys/clob/params";
+    option (google.api.http).get = "/elys-network/elys/clob/sub-accounts";
   }
 
   rpc MarketOrders(MarketOrdersRequest) returns (MarketOrdersResponse) {
-    option (google.api.http).get = "/elys-network/elys/clob/all-markets";
+    option (google.api.http).get = "/elys-network/elys/clob/market-orders";
   }
 
   rpc OwnerPerpetuals(OwnerPerpetualsRequest)
       returns (OwnerPerpetualsResponse) {
-    option (google.api.http).get = "/elys-network/elys/clob/all-markets";
+    option (google.api.http).get = "/elys-network/elys/clob/owner-perpetuals";
+  }
+
+  rpc PerpetualOrder(PerpetualOrderRequest)
+      returns (PerpetualOrderResponse) {
+    option (google.api.http).get = "/elys-network/elys/clob/perpetual-order";
+  }
+
+  rpc AllPerpetualOrder(AllPerpetualOrderRequest)
+      returns (AllPerpetualOrderResponse) {
+    option (google.api.http).get = "/elys-network/elys/clob/all-perpetual-order";
+  }
+
+  rpc CurrentTwapPrice(CurrentTwapPriceRequest)
```

Comment:
Add query for user perpetual order, it will use OrderOwner to fetch order keys and then list of current open orders